sites/next.skeleton.dev/src/content/docs/tutorials/figma/chapter-1.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: Chapter 1 - Setup & Usage
+description: Learn how to set up and use the Skeleton Figma Design System.
+order: 0
+---
+
+export const components = componentSet;
+
+import Process from '@components/docs/Process.astro';
+import ProcessStep from '@components/docs/ProcessStep.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
+
+import { Image } from 'astro:assets';
+import SvgSkeleton from '@images/tutorials/figma/skeletonFileIcon.svg';
+import ImgCreateNewProject from '@images/tutorials/figma/chapter-1/01_createNewProject.webp';
+import ImgImportProject from '@images/tutorials/figma/chapter-1/02_importProject.webp';
+import ImgPublish from '@images/tutorials/figma/chapter-1/03_publish.webp';
+import ImgCreateNewFile from '@images/tutorials/figma/chapter-1/04_createNewFile.webp';
+import ImgSelectLibrary from '@images/tutorials/figma/chapter-1/05_selectLibrary.webp';
+import ImgUseLibrary from '@images/tutorials/figma/chapter-1/06_useLibrary.webp';
+
+---
+
+## How to Install the Figma Library
+
+For best results, a <a href="https://www.figma.com/pricing/#figma" class="chip preset-filled mx-1" target="_blank">Figma Professional Team Plan</a> (or higher) is recommended.
+
+<Process>
+    <ProcessStep step="1">
+        ### Purchase the Figma Asset
+
+        <div class="card-enhanced border-[1px] border-surface-200-800 space-y-2 p-4 md:p-8 flex flex-row justify-between items-center">
+            <div className="flex flex-row gap-4 items-center">
+            <Image src={SvgSkeleton} class="rounded-container" alt="Skeleton" />
```

Comment:
The SVG image here gets a bit lost. It's very detailed for it's size. Maybe try scaling that back to just a simple document image with the Skull and Figma icons?